lisa/operating_system.py

```diff
@@ -289,9 +530,33 @@ def name_pattern(cls) -> Pattern[str]:
     def _initialize_package_installation(self) -> None:
         self._node.execute("zypper --non-interactive --gpg-auto-import-keys update")
 
-    def _install_packages(self, packages: Union[List[str]]) -> None:
-        command = f"zypper --non-interactive in  {' '.join(packages)}"
-        self._node.execute(command, sudo=True)
+    def _install_packages(
+        self, packages: Union[List[str]], signed: bool = True
+    ) -> None:
+        command = f"zypper --non-interactive in {' '.join(packages)}"
+        if not signed:
+            command.__add__(" --no-gpg-checks")
+        install_result = self._node.execute(command, sudo=True)
+        if install_result.exit_code in (1, 100):
+            raise LisaException(
+                f"Failed to install {' '.join(packages)}."
+                f" stdout: {install_result.stdout}"
```

Comment:
It doesn't need to output the stdout, unless there is specified error message. The stdout may be many lines, it's not friendly.

Reply:
replacing with stderr

Reply:
make sure those error messages helpful, and not too many.